cmd/gokr-stat/stat.go

```diff
@@ -85,21 +87,42 @@ func printStats() error {
 	type processAndFormatter interface {
 		ProcessAndFormat(map[string][]byte) []stat.Col
 	}
-	modules := []processAndFormatter{
-		&cpu.Stats{},
-		&disk.Stats{},
-		&sys.Stats{},
-		&net.Stats{},
-		&mem.Stats{},
+
+	var modules []processAndFormatter
+
+	if *thermalFlag {
+		modules = []processAndFormatter{
+			&cpu.Stats{},
+			&disk.Stats{},
+			&sys.Stats{},
+			&net.Stats{},
+			&mem.Stats{},
+			&thermal.Stats{},
+		}
+	} else {
+		modules = []processAndFormatter{
+			&cpu.Stats{},
+			&disk.Stats{},
+			&sys.Stats{},
+			&net.Stats{},
+			&mem.Stats{},
+		}
 	}
+
 	header := func() {
 		const blue = "\033[1;34m"
 		fmt.Printf(blue + "usr sys idl wai stl | ")
 		fmt.Printf(" read  writ | ")
 		fmt.Printf(" int   csw  | ")
 		fmt.Printf(" recv  send | ")
-		fmt.Printf(" used  free  buff  cach\n")
+		if *thermalFlag {
+			fmt.Printf(" used  free  buff  cach | ")
+			fmt.Printf(" tz0\n")
```

Comment:
I wonder if we can make it “cpu” instead of tz0. At least on the Pi, that should be the case.